pkg/summary/collector.go

```diff
@@ -108,6 +108,7 @@ func (tc *logCollector) CollectSuccessUnit(name string, unitCount int, arg inter
 			tc.successUnitCount += unitCount
 		} else {
 			tc.successCosts[name] += v
+			tc.successUnitCount += unitCount
```

Comment:
so I wonder why can't we just replace the entire lines 106 to 112 by

```go
tc.successCosts[name] += v
tc.successUnitCount += unitCount
```

similar for lines 114 to 118:

```go
tc.successData[name] += v
```